documentation/asciidoc/computers/configuration/display-resolution.adoc

```diff
@@ -18,69 +18,87 @@ Flagship models since Raspberry Pi 4B, Compute Modules since CM4 (except for CM4
 ----
 $ kmsprint | grep Connector
 ----
 
-==== Set a custom resolution
+What this command does
+
+- kmsprint shows display information
+- grep Connector filters the output to show only display connectors
 
-To set a custom resolution, use our Screen Configuration tool, `raindrop`. If your Raspberry Pi OS installation doesn't already include `raindrop` (for instance, if you're still using the previous Screen Configuration tool, `arandr`), you can download `raindrop` from `apt` or the Recommended Software GUI.
+This helps identify connected monitors for troubleshooting purposes.
 
-==== Set a custom rotation
+=== Notes about older screen configuration tools
 
-To set a custom resolution, use our Screen Configuration tool, `raindrop`. If your Raspberry Pi OS installation doesn't already include `raindrop` (for instance, if you're still using the previous Screen Configuration tool, `arandr`), you can download `raindrop` from `apt` or the Recommended Software GUI.
+Earlier versions of Raspberry Pi OS used different utilities for display configuration, such as:
 
-If you run the Wayland desktop compositor, you can set a custom display rotation with `wlr-randr`. The following commands rotate the display by 0°, 90°, 180°, and 270°:
+- Screen Configuration
+
+- raindrop
+
+- arandr
+
+These tools are no longer used in Raspberry Pi OS Trixie.
+You do not need to install or remove them, and any documentation that refers to these tools should be considered outdated.
+
+For current systems, always use the Control Centre for display configuration.
+
+
+[TIP]
+====
+If you have already installed `raindrop`, you can uninstall it with the following command:
 
 [source,console]
 ----
-$ wlr-randr --output HDMI-A-1 --transform normal
-$ wlr-randr --output HDMI-A-1 --transform 90
-$ wlr-randr --output HDMI-A-1 --transform 180
-$ wlr-randr --output HDMI-A-1 --transform 270
+$ sudo apt remove --purge raindrop -y
 ----
 
-The `--output` option specifies the device to be rotated.
+To uninstall `arandr`, run the following command:
+
+[source,console]
+----
+$ sudo apt purge arandr
+----
+====
 
-NOTE: To run this command over SSH, add the following prefix: `WAYLAND_DISPLAY=wayland-1`, e.g. `WAYLAND_DISPLAY=wayland-1 wlr-randr --output HDMI-A-1 --transform 90`.
+=== Summary
 
-You can also use one of the following `--transform` options to mirror the display at the same time as rotating it: `flipped`, `flipped-90`, `flipped-180`, `flipped-270`.
+- Use Control Centre to change resolution and rotation
 
-=== Console resolution and rotation
+- Use the terminal only for checking display information
 
-To change the resolution and rotation of your Raspberry Pi in console mode, use the KMS settings. For more information, see <<kernel-command-line-cmdline-txt,configuring the kernel command line>>.
+- Ignore old tutorials that mention raindrop or arandr
 
-NOTE: When using console mode with multiple displays, all connected displays share the same rotation settings.
```

Comment:
Please don't remove this section, as it's still useful for users of Raspberry Pi OS Lite.